packages/components/src/ThumbnailGallery/ThumbnailGallery.css

```diff
@@ -0,0 +1,9 @@
+.thumbnail {
+  display: inline-block;
```

Comment:
I haven't played around with this yet, but I wonder if a `display: flex` on the parent might suit our needs better

I'll update once I've actually tried it out